content/en/synthetics/multistep.md

```diff
@@ -33,16 +35,22 @@ Multistep API tests can run from Datadog [managed locations][4] and [private loc
 
 **Note**: Multistep API tests allow you to link several HTTP requests in a single test. If you want to perform single requests to your services, you can leverage [API tests][6].
 
+## Permissions
+
+By default, only users with the [Datadog Admin and Datadog Standard roles][7] can create, edit, and delete Synthetic multistep API tests. To get create, edit, and delete access to Synthetic multistep API tests, upgrade your user to one of those [default roles][7].
+
+If you have access to the [custom role feature][8], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.
+
 ## Configuration
 
 ### Name and tag your test
 
-1. **Name** your Multistep API test.
-2. Add `env` **Tags** as well as any other tag to your Multistep API test. You can then use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][7].
+1. Name your Multistep API test.
+2. Add `env` and other tags to your Multistep API test. Then, use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][9].
```

Comment:
It feels like it's something users would want to do right after adding those tags. The second sentence is actually more about explaining why tags are useful.

Reply:
```suggestion
2. Add `env` and other tags to your Multistep API test. You can use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][9].
```